scriptmodules/helpers.sh

```diff
@@ -261,6 +261,10 @@ function _mapPackage() {
                         isPlatform "rpi5" && pkg="linux-headers-rpi-2712"
                     fi
                 fi
+            elif isPlatform "armbian"; then
+                local branch="$(sed -n '/^BRANCH/ s/.*=//p' /etc/armbian-release)"
+                local family="$(sed -n '/^LINUXFAMILY/ s/.*=//p' /etc/armbian-release)"
```

Comment:
Not much simpler but I think I would have gone for `grep -oP "LINUXFAMILY=\K.*"`

Reply:
(We use this elsewhere a fair amount)

Reply:
:-)